stytra/experiments/__init__.py

```diff
@@ -365,15 +365,32 @@ def save_data(self):
                 version = None
 
                 try:
-                    repo = git.Repo(sys.argv[0], search_parent_directories=True)
-                    git_hash = repo.head.object.hexsha
+
+                    ####################### try get repo even during testing ###############################
+                    if (
+                        "pytest" in sys.argv[0]
+                    ):  # if first element points to pytest, check other addresses in args
+
+                        for el in sys.argv:
+                            if os.path.isdir(el):
+                                repo = git.Repo(el, search_parent_directories=True)
+                                break
+
+                    else:
+
+                        repo = git.Repo(sys.argv[0], search_parent_directories=True)
+                        git_hash = repo.head.object.hexsha
+                    #########################################################################################
```

Comment:
Updated repo when looking it from pytests context (added break in the loop as suggested previously [here](https://github.com/portugueslab/stytra/pull/62/files#r748730106)